charts/portal/README.md

```diff
@@ -66,6 +66,8 @@ This section describes configurable parameters in **values.yaml**, there is also
 | `global.databasePassword` | Database password | `7layer` |
 | `global.databaseHost` | Database Host | `` |
 | `global.databasePort` | Database Port | `3306` |
+| `global.databaseUseSSL` | use SSL when communicating with the Database | true |
```

Comment:
First letter capital and value should be in quotes.

Reply:
fixed